pocket_coffea/lib/leptons.py

```diff
@@ -17,9 +17,7 @@ def lepton_selection(events, lepton_flavour, params):
         # Requirements on SuperCluster eta, isolation and id
         etaSC = abs(leptons.deltaEtaSC + leptons.eta)
         passes_SC = np.invert((etaSC >= 1.4442) & (etaSC <= 1.5660))
-        passes_iso = True
-        if "iso" in cuts.keys():
-            passes_iso = leptons.pfRelIso03_all < cuts["iso"]
+        passes_iso = leptons.pfRelIso03_all < cuts["iso"]
```

Comment:
Hi! I just have a question about this change.  Not in all the analysis this "iso" key is used, that's why there was this if clause. Does that cause problems to you ? 

We need to keep the function general enough to support different data-taking periods configuration. Alternatively we can create different functions

Reply:
Actually not, we can also keep general and stay with an if cause

Reply:
Ehi @ValVau if you push a commit to keep the if I think the PR is good to go!

Reply:
Grazie mille. I pushed a commit keeping the if. Please take a look.